wca-regulations.md

```diff
@@ -236,6 +236,7 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
     - 9f1) All timed results under 10 minutes, except for 3x3x3 Multi-Blind, are measured and truncated to the nearest hundredth of a second. All timed averages and means under 10 minutes are measured and rounded to the nearest hundredth of a second.
     - 9f2) All timed results, averages, and means over 10 minutes, as well as all times for 3x3x3 Multi-Blind results, are measured and rounded to the nearest second (e.g. x.49 becomes x, x.50 becomes x+1).
     - 9f4) The result of an attempt is recorded as DNF (Did Not Finish) if the attempt is disqualified or unsolved/unfinished.
+        - 9f4a) If a competitor violates a regulation that would result in disqualification of their attempt, the attempt should be stopped immediately after this violation.
```

Comment:
```suggestion
        - 9f4a) If a competitor violates a regulation that would result in disqualification of their attempt, the attempt should be stopped immediately after this violation, at the discretion of the judge or WCA Delegate.
```

Reply:
@Nevseros how does this sound? can remove clearly here

Reply:
LGTM!

Reply:
I'd edit a little further and provide more specific guidance, to reduce ambiguity.
```suggestion
        - 9f4a) If a competitor violates a regulation clearly enough that the attempt is certain to be disqualified, the judge should immediately stop the attempt. If the judge is uncertain or a dispute could be disruptive (e.g. because an extra attempt could delay the competition), the judge should consult the WCA Delegate.
```

Reply:
I think that works far better